README.md

```diff
@@ -128,6 +128,11 @@ $ pip uninstall hokusai
 $ brew link --overwrite hokusai
 ```
 
+If you are on Linux:
+```
+curl -sSL https://raw.githubusercontent.com/artsy/hokusai/master/get-hokusai.sh | sudo bash
```

Comment:
**nit (non-blocking)**: FYI, there's also a `docs` commit type (vs `chore`) commonly used documentation updates like this.

Didn't know we had this handy install script! 🎉

Reply:
ooo will use `docs` next time :)